pyproject.toml

```diff
@@ -46,6 +47,9 @@ requires-python = ">=3.8"
 [project.entry-points.pytest11]
 conda-solvers = "pytest_conda_solvers.plugin"
 
+[project.scripts]
+pcs-tool = "pytest_conda_solvers.cli:app"
```

Comment:
How often do we expect to use this? If it's just a dev tool, could we get away with just `python -m pytest_conda_solvers.cli` or `python pytest_conda_solvers/cli.py`. That's what we do, e.g. in menuinst or conda-smithy, for schema regeneration.

Reply:
Yeah, I can get on board with that. Perhaps I'll add it as a pixi task as well.